x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -48,57 +75,58 @@ const Container = styled<
   }
 `;
 
-interface TimelineMargins {
+interface ITimelineMargins {
   right: number;
   left: number;
   top: number;
   bottom: number;
 }
 
-interface Props {
-  timelineMargins: TimelineMargins;
+interface IWaterfallItemProps {
+  timelineMargins: ITimelineMargins;
   totalDuration: number;
   item: IWaterfallItem;
   color: string;
   isSelected: boolean;
   onClick: () => any;
 }
 
+function Prefix({ item }: { item: IWaterfallItem }) {
```

Comment:
Should we call it `TransactionPrefixIcon` just to be super clear?

Reply:
I have a feeling which ones get prefixed and with what is going to change a bunch and this is a "private" component. Could be "WaterfallItemPrefix" if we are worried about stack traces maybe?

Reply:
Not following you completely. Are you saying this is currently an icon that prefixes Transactions, but that will change in the future? Or what would you say it does?